[MIG] account_background_post: Migration to 19.0

migration-bot-adhoc opened the pull request.

migration-bot-adhoc committed:
[MIG] account_background_post: Migration to 19.0

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-invoicing/pull/221.png)](https://mergebot.adhoc.ar/ingadhoc/account-invoicing/pull/221)

cav-adhoc committed:
[IMP] account_background_post: refactor of background post cron job

cav-adhoc committed:
[FIX] account_background_post: correctly count invoices to post in background
- Add field move_ids to default_get
- Fix condition to check if only one invoice is being posted

rov-adhoc commented:
@roboadhoc r+

roboadhoc commented:
@migration-bot-adhoc @rov-adhoc because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

rov-adhoc commented:
@roboadhoc r+ rebase-ff

roboadhoc commented:
This PR is already reviewed, reviewing it again is useless.

roboadhoc commented:
Merge method set to rebase and fast-forward.
